autogpt/llm/providers/openai.py

```diff
@@ -0,0 +1,40 @@
+from autogpt.llm.base import ModelInfo, ModelType
+
+OPEN_AI_CHAT_MODELS = {
```

Comment:
❤️ theres a million places this will be useful! forgive me but does this cover embed_dims in this pr?

Reply:
Will add to the struct

Reply:
Should it was the question sorry, not the most familiar with that section of code